src/test/java/net/bramp/ffmpeg/FFprobeTest.java

```diff
@@ -57,6 +58,13 @@ public void testVersion() throws Exception {
   }
 
   @Test
+  public void testStartPtsType()
```

Comment:
This isn't the way I suspected you would check pts was a long. This tests how the code looks, now how it actually works.

Instead, I suggest updating 'fixtures/ffprobe-Always On My Mind [Program Only] - Adelén.mp4', to have a start_pts that is longer than a int can handle. Then test for the correct value in say FFprobeTest .testProbeVideo2.

Or, even better, take a ffprobe of your test file that exposed this problem in the first place, and add a new fixture, and new test case.